container/buildDockerImage.sh

```diff
@@ -7,15 +7,29 @@ set -o pipefail
 SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 IMAGES_DIR="${SCRIPT_CURRENT_DIR}/images"
 DEFAULT_IMAGE_PREFIX="accelerate-io/dpnt-coverage-"
-BASE_IMAGE_VERSION=$( cat "${IMAGES_DIR}/base/version.txt" | tr -d " " | tr -d "\n" )
-BASE_IMAGE_TAG="${DEFAULT_IMAGE_PREFIX}base:${BASE_IMAGE_VERSION}"
+
+BASE="base"
+LANGUAGE_ID=$1
+case ${LANGUAGE_ID} in
+    csharp|fsharp|vbnet)
+    BASE="dotnet" ;;
+esac
+
+BASE_IMAGE_VERSION=$( cat "${IMAGES_DIR}/${BASE}/version.txt" | tr -d " " | tr -d "\n" )
+ROOT_BASE_IMAGE_TAG="${DEFAULT_IMAGE_PREFIX}base:${BASE_IMAGE_VERSION}"
+DOTNET_BASE_IMAGE_TAG="${DEFAULT_IMAGE_PREFIX}dotnet:${BASE_IMAGE_VERSION}"
+BASE_IMAGE_TAG=${ROOT_BASE_IMAGE_TAG}
 
 function die() { echo >&2 $1; exit 1; }
 [ "$#" -eq 1 ] || die "Usage: $0 LANGUAGE_ID"
-LANGUAGE_ID=$1
 
 echo "~~~~~~ Refreshing base image ~~~~~~"
-docker build -t ${BASE_IMAGE_TAG} "${IMAGES_DIR}/base/."
+if [[ "${BASE}" == "base"  ]]; then
+    docker build -t ${BASE_IMAGE_TAG} "${IMAGES_DIR}/${BASE}/."
```

Comment:
You still need to refresh the base, even for dotnet, as the dotnet-base depends on base.